resources/views/includes/sidebar.blade.php

```diff
@@ -9,7 +9,8 @@
                 </a>
             </li>
             <li>
-                <a href="">
+                <!-- redirects to profile page based on username -->
+                <a href="profile/{{ Auth::user()->username }}">
```

Comment:
can we safeguard the route by adding a dot in the beginning. this should be: "./profile/{{ Auth::user()->username }}"